sw/device/tests/usbdev_stream_test.c

```diff
@@ -35,10 +35,11 @@
 #define NUM_STREAMS USBUTILS_STREAMS_MAX
 #endif
 
+#define TRANSFER_BYTES_SILICON (0x10U << 20)
 // This takes about 256s presently with 10MHz CPU in CW310 FPGA and physical
 // USB with randomized packet sizes and the default memcpy implementation;
 // The _MEM_FASTER switch drops the run time to 187s
-#define TRANSFER_BYTES_FPGA (0x10U << 20)
+#define TRANSFER_BYTES_FPGA (0x10U << 16)
```

Comment:
The comment needs modifying since it relates to the _FPGA definition. Suggest replacing 'This takes' with '16MiB takes..' and then extending the comment with an annotation that the transfer size has been reduced to keep CI jobs completing in under 30s.